hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/TestAlterTableDropPartition.scala

```diff
@@ -265,53 +187,4 @@ class TestAlterTableDropPartition extends TestHoodieSqlBase {
       }
     }
   }
-
-  Seq(false, true).foreach { hiveStyle =>
-    test(s"Purge drop multi-level partitioned table's partitions, isHiveStylePartitioning: $hiveStyle") {
```

Comment:
the deleted UTs are now obsolete since it's done by cleaner async, previously it's fs.delete. @nsivabalan i think with manual testing passing for the commands we should be good to go. UT can be a follow-up task

Reply:
sure, sg

Reply:
Let's make sure there's a ticket for this

Reply:
this is fixed now with forcing clean